ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -872,6 +872,33 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-stack-upi
+- as: azure-ipi-fullyprivate-proxy-f360-compliance-operator
+  cron: 22 9 24 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      FIPS_ENABLED: "true"
+      TEST_FILTERS: ~HyperShiftMGMT&;Compliance_Operator&;~Disruptive&
+      TEST_PARALLEL: "3"
+      TEST_SCENARIOS: Compliance_Operator
+    test:
+    - ref: openshift-extended-test
+    workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
+- as: azure-ipi-fullyprivate-proxy-f360-compliance-operator-destru
```

Comment:
Typo issue. should be `destructive`

Reply:
It is on purpose. Otherwise `make update` will report below error:
`time="2024-10-22T00:50:59Z" level=error msg="Failed to load CI Operator configuration" error="invalid ci-operator config: invalid configuration: tests[11].as: 65 characters long, maximum length is 61" source-file=/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable.yaml`

Reply:
I was assuming as typo issue. Thank you for confirming.